frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -162,3 +165,12 @@ export const removeTopologyResourceConnection = (edge: Edge): Promise<any> => {
 
   return removeResourceConnection(source, target);
 };
+
+export const getSeverityAlertType = (alerts: Alert[]): AlertSeverity => {
+  const sortedAlerts = sortMonitoringAlerts(alerts);
+  const severityType = sortedAlerts[0]?.labels.severity as AlertSeverity;
+  return severityType;
```

Comment:
Has potential to be `undefined` based on `sortedAlerts[0]?`. Should there be a fallback?